From a6b4e5e0a8f1aebf345badb8c7e6773d45895e38 Mon Sep 17 00:00:00 2001
From: 115100 <git@three-sigma.org>
Date: Wed, 22 Oct 2025 12:46:53 +0100
Subject: [PATCH] cmake: add CEF lib directory to install RPATH

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9a728c1..1f632a6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -506,6 +506,9 @@ if(UNIX AND NOT APPLE)
     if(TARGET CEF::Library)
         message(STATUS "Linking with FindPackage version of CEF::Library")
         target_link_libraries(bolt PUBLIC CEF::Library)
+        if(NOT BOLT_SKIP_RPATH)
+            set_target_properties(bolt PROPERTIES INSTALL_RPATH_USE_LINK_PATH TRUE)
+        endif()
     else()
         target_link_libraries(bolt PUBLIC "${BOLT_LIBCEF_DIRECTORY}/libcef.so")
     endif()
-- 
2.51.0

